locales/en-US/server.properties

```diff
@@ -0,0 +1,150 @@
+# global phrases shared across pages, prefixed with 'g'
+
+# TODO: replace each page's reference to this with the global
+gMyShots = My Shots
+
+# creating page
+
+# Localization note: {title} is a placeholder for the title of the web page
+# captured in the screenshot. The default, for pages without titles, is
+# creatingPageTitleDefault.
+creatingPageTitle = Creating {title}
+creatingPageTitleDefault = page
+
+# home page
+
+# Note: do not translate the phrase 'Firefox Screenshots'.
+homePageDescription = Share anything on the web with anyone using Firefox Screenshots.
+# TODO: do we need this separate description meta tag?
+homePageDescription2 = Intuitive screenshots baked right into the browser. Capture, save and share screenshots as you browse the Web using Firefox.
+homePageButtonMyShots = Go To My Shots
+homePageTitleBeta = Beta
+homePageTeaser = Coming Soon...
+
+# leave screenshots page
+
+leavePageConfirmDelete = Confirm account deletion
+leavePageErrorAddonRequired = You must have the addon installed to delete your account
```

Comment:
Not a matter of preference, addon is the wrong spelling ;-)
https://transvision.mozfr.org/?recherche=add-on&repo=central&sourcelocale=en-US&locale=it&search_type=strings_entities